test/intl402/supportedLocalesOf-returned-array-elements-are-not-frozen.js

```diff
@@ -5,14 +5,20 @@
 es5id: 9.2.8_4
 description: >
     Tests that the array returned by SupportedLocales is extensible,
-    but its properties are non-writable/non-configurable.
+    writable and configurable.
 author: Norbert Lindenberg
 includes: [testIntl.js]
 ---*/
 
-function testFrozenProperty(obj, property) {
+function testNormalProperty(obj, property) {
     var desc = Object.getOwnPropertyDescriptor(obj, property);
-    assert.sameValue(desc.writable, false, "Property " + property + " of object returned by SupportedLocales is writable.");
+    assert.sameValue(desc.writable, true, "Property " + property + " of object returned by SupportedLocales is not writable.");
+    assert.sameValue(desc.configurable, true, "Property " + property + " of object returned by SupportedLocales is not configurable.");
+}
+
+function testLengthProperty(obj, property) {
+    var desc = Object.getOwnPropertyDescriptor(obj, property);
+    assert.sameValue(desc.writable, true, "Property " + property + " of object returned by SupportedLocales is not writable.");
```

Comment:
the negative messages for the assertions are really confusing. I see that `desc.writable` should be `true` and then a message saying it's not writable. Assertions messages shouldn't be error messages, but a description of each assertion.

This doesn't need to be changed in this PR, as this file already exists, but it would be nicer in the future PRs to follow a convention.